### ISSUE TITLE
How to plot epipolar lines with the existing code?

### ISSUE THREAD

**Issue by LavenderLA** [dcharatan/pixelsplat#14]
Hi, thanks for your wonderful work and extraordinary codebase! I also want to implement the code with my own dataset. Would you please kindly share an example of how to plot epipolar lines with the existing code like what you've mentioned in this [issue](https://github.com/dcharatan/pixelsplat/issues/4#issuecomment-1875738609)? It would be really helpful since I want to check whether my input data format is correct.

Thanks so much!

**Comment by dcharatan** [comment]
I added a script at `src/scripts/visualize_epipolar_lines.py` that shows an example of how to do this. It reads the dataset configuration from the main configuration file and prints a few epipolar lines to a folder called `epipolar_lines`. I also added a VS Code launch configuration for the script that shows how to run it. Here's an example output:

![example_03](https://github.com/dcharatan/pixelsplat/assets/13124225/edb41133-3415-4cc2-8713-1064a2f64d2b)

Hope this helps!
